docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+   +------------------------+---------------------------------------------------------------------------------+
+   | inland                 | Creates an inland land mask by determining in-land (i.e. non-coastal) points    |
+   |                        | and assigning a value of 1. Default value is 0.                                 |
+   +------------------------+---------------------------------------------------------------------------------+
+   | orog_gsl               | Ceates orographic statistics fields required for the orographic drag suite      |
+   |                        | developed by NOAA's Global Systems Laboratory (GSL)                             |
+   +------------------------+---------------------------------------------------------------------------------+
+   | fregrid                | Remaps data from the input mosaic grid to the output mosaic grid                |
+   +------------------------+---------------------------------------------------------------------------------+
+   | lakefrac               | Calculates the ratio of the lake area to the grid cell area at each atmospheric |
+   |                        | grid point.                                                                     |
+   +------------------------+---------------------------------------------------------------------------------+
+
+.. _Data:
+
+Download and Stage the Data
+============================
+
+The SRW App requires input files to run. These include static datasets, initial and boundary conditions files, and model configuration files. On Level 1 and 2 systems, the data required to run SRW App tests are already available. For Level 3 and 4 systems, the data must be added. Detailed instructions on how to add the data can be found in the :numref:`Section %s Downloading and Staging Input Data <DownloadingStagingInput>`. :numref:`Sections %s <Input>` and :numref:`%s <OutputFiles>` contain useful background information on the input and output files used in the SRW App. 
+
+.. _GridSpecificConfig:
+
+Grid Configuration
+=======================
+
+The SRW App officially supports three different predefined grids as shown in :numref:`Table %s <PredefinedGrids>`. The "out-of-the-box" SRW App case uses the ``RRFS_CONUS_25km`` predefined grid option. More information on the predefined and user-generated grid options can be found in :numref:`Chapter %s <LAMGrids>` for those who are curious. Users who plan to utilize one of the three pre-defined domain (grid) options may continue to :numref:`Step %s <GenerateForecast>`. Users who plan to create a new domain should refer to :numref:`Chapter %s <LAMGrids>` for details on how to do so. At a minimum, these users will need to add the new grid name to the ``valid_param_vals`` script and add the corresponding grid-specific parameters in the ``set_predef_grid_params`` script. 
+
+.. _PredefinedGrids:
+
+.. table::  Predefined grids in the SRW App
```

Comment:
Will user generated grids be fully supported? That's great, if so! I think we are at the point of being able to do that for this second release. If not, I would clearly state that the predefined domains are the fully supported domains in the release.

Reply:
I think they will be supported, but not _fully_ supported, but I'll let @JeffBeck-NOAA and @mark-a-potts weigh in.